### ISSUE TITLE
Support for HTML documents?

### ISSUE THREAD

**Issue by sdwarwick** [Jason-Rev/vscode-spell-checker#37]
I don't seem to see support for HTML documents.. perhaps the most needed?  I've tried adding "html" or "HTML" to the supported programming lists to no avail..

**Comment by Jason3S** [comment]
Did you add it to your user settings like this:

```javascript
    // Specify file types to spell check.
    "cSpell.enabledLanguageIds": [
        "csharp",
        "go",
        "html",                   // <-- here
        "javascript",
        "javascriptreact",
        "markdown",
        "php",
        "plaintext",
        "python",
        "text",
        "typescript",
        "typescriptreact",
        "yml"
    ]
```

You then need to restart VS Code.  For some reason the change is not taking without a reset.  I'll open a bug on that.

**Comment by sdwarwick** [comment]
that was it!    actually added "html" at the end..  

enhancement requests:
 - right click on mouse to provide an "add to local dictionary"
 - path to global dictionary in docs

 - The only issue I've got with this for a programming speller is that it finds all the identifier names as well.  Golang has an extensive system for dealing with identifiers already.   wondering if something that limits the spellcheck to only things in "" quotes might be a good idea for specific languages??

**Comment by Jason3S** [comment]
I am going to close this as fixed.

#13 Addresses being able to turn on/off a file type.  It will most likely be by clicking on the spell checker status bar item.  It is not yet possible to add things to the right click menu.

#3 Addresses the issue of only checking String.
